### PR TITLE
Convert Use Cases from google docs to markdown.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #46.

This is the contents of the current draft of the [SDS Use Cases Document](https://docs.google.com/document/d/18UdEjA2kQRnILnBEnWijNho0YMJ93dbjKntx0NQzapI/edit#). It does not include the sidebar comments (so please review and comment on this PR as necessary).

### CODE REVIEW

File: use_cases.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+## ​1.7.​ Provide a mechanism for receiving and handling a queue of task-based messages.
+
+As a user, I would like a mechanism where other entities can send me actionable
+message objects that are synced across my SDS instances, so that I can act on
+them in whatever way is appropriate for the type of message. If subsequent
+messages are exchanged as a result of an initiating message, the subsequent
+messages should be referentially tied together (e.g. thread back to the parent).
+
+## ​1.8.​ Alice wants to upload executable code or configuration management that service providers will run based on an event system.
```
Comment (dmitrizagidulin) [inline use_cases.md:77]: I think this may be out of scope of this Working Group.

Comment (csuwildcat) [reply use_cases.md:77]: Yes, I believe it is out of scope - can we remove it @OR13?

File: use_cases.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+Ensure users have the ability to terminate a relationship with one or more
+service providers in a manner that does not disclose anything about the future
+state or location of the user’s data. Ensure the user can verify or obtain a
+verifiable certification from service provider that their data has been, if
+requested, purged from the provider's system (confirming right to be forgotten).
+
+## 1.10 Sharding data
+
+Ensure users have the ability to shard their data, or create M of N key
```
Comment (dmitrizagidulin) [inline use_cases.md:95]: I'm not sure what this section means, may need to be rephrased.

Comment (csuwildcat) [reply use_cases.md:95]: This feels a little too specific, but we should probably ask whoever wrote it if it is something they feel strongly about.

**Comment by ewelton** [comment]
Please also address issue #50

**Comment by dmitrizagidulin** [comment]
@ewelton 
> Please also address issue #50

No worries :) There's a tracking issue for it, it won't get lost.

File: use_cases.md (hunk 1 above)
Comment (OR13) [reply use_cases.md:77]: @csuwildcat not sure how you plan to handle complex agent-like functionality without creating configuration hell, or map / reduce / event handlers, but I would be happy to see that section gone.

File: use_cases.md (hunk 2 above)
Comment (OR13) [reply use_cases.md:95]: I think this is an app level thing, not an edv base layer thing... no reason to remove it yet though.

**Comment by OR13** [comment]
Lets get this merged, there appears to be a desire to move sections in and out of it, and I would like to see smaller PRs that do that next.

File: use_cases.md (hunk 1 above)
Comment (csuwildcat) [reply use_cases.md:77]: We never needed that in our model, so I am not sure what you mean. The Hub never processed the feed, it pushed it to subscribed entities (your wallet, people you allow to see certain Actions, etc.) who had signed capability access, and they processed it externally.

Perhaps I can approve this and then file to remove it.

**Comment by OR13** [comment]
I'm gonna merge this, so that we can take PRs to slice it up

**Pull request merged**